app/models/album.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const errors = require('../errors');
+
+module.exports = (sequelize, DataTypes) => {
+  const userAlbum = sequelize.define('user_album', {
```

Comment:
Check properties definition for table names http://docs.sequelizejs.com/manual/tutorial/models-definition.html#configuration

Reply:
`underscored: true` and `freezeTableName: true`